src/evict/evict_inline.h

```diff
@@ -258,9 +361,17 @@ __wt_evict_page_cache_bytes_decr(WT_SESSION_IMPL *session, WT_PAGE *page)
 }
 
 /*
+ * NO_AUTO_FORMAT
+ *
  * __wt_evict_clean_pressure --
- *     Return true if clean cache is stressed and will soon require application threads to evict
- *     content.
+ *     This function checks whether the cache is approaching or has surpassed its eviction trigger
+ *     thresholds, indicating that application threads will soon be required to assist with
+ *     eviction.
+ *
+ *     At present, this function is primarily called by the prefetch thread to determine whether it
+ *     should avoid to prefetch pages, as application threads may soon be involved in eviction.
```

Comment:
```suggestion
 *     should avoid prefetching pages, as application threads may soon be involved in eviction.
```

Reply:
updated in [73744c3](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/73744c3a2b3e6a9811054165f507c2971025785e)